### ISSUE TITLE
Add note that helper and tooltip of CBG and RBG are not reliably announced by screen readers

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#2607]
Some browser and screen readers combinations (e.g. NVDA on Chrome) do not announce helpers and tooltips on CheckboxGroup and RadioButtonGroup (because they ignore the `aria-describedby` attribute on elements with `role="group"`). Add a note about the unreliable support in screen readers of helpers and tooltips on these components.

---

If you think this issue is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.
